go/api_default.go

```diff
@@ -67,63 +69,99 @@ func handleNotFoundError(w http.ResponseWriter, detail string) {
 	w.Write(payload)
 }
 
-func callGrafana(url string, apiKey string, verb string, payload io.Reader) (int, []byte, error) {
+func callGrafana(admin bool, url string, auth string, verb string, payload io.Reader) (int, []byte, error) {
 	var statusCode int
 	var body []byte
 	var err error
+
+	if os.Getenv("DEBUG") == "true" {
```

Comment:
So it's probably easier to use the inbuilt way of doing this ...

```Go
# either take the debug option off the cli args or the environment i.e. 
cli.StringFlag{
    Name:   "debug",
    Usage:  "Used to enabled debug logging` BOOLEAN`",
    EnvVar: "DEBUG",
},
...
if ctx.Boolean("debug") {
   logrus.SetLevel(logrus.DebugLevel)
}
..
logrus.Debugf("blah bah")
```



```

Reply:
I thought this would be a thing yeh. Thanks.